tests/test_masking.py

```diff
@@ -26,6 +26,28 @@ def setUp(self):
             self.imlist.append(im)
         self.stack = ImageStack(self.imlist)
 
+    def test_apply_trajectory_mask(self):
+        starting_pixel = (20,20) 
+        velocity = (50.0, -10.0)
+        radius = 5.0
+        dt = self.stack.get_single_image(1).get_obstime() - self.stack.get_single_image(0).get_obstime()
+        trj = make_trajectory(starting_pixel[0], starting_pixel[1], velocity[0], velocity[1])
+        bitmask = 1
+
+        masked_stack = mask_trajectory(trj, self.stack, radius, bitmask)
+
+        for i in range(self.img_count):
+            origin_of_mask = (starting_pixel[0] + velocity[0] * i * dt , starting_pixel[1] + velocity[1] * i * dt)
+            msk = masked_stack.get_single_image(i).get_mask()
+        
+            for x in range(self.dim_x):
+                for y in range(self.dim_y):
+                    distance = (x - origin_of_mask[0])**2 + (y - origin_of_mask[1])**2
+                    if distance <= radius**2:
+                        self.assertEqual(msk.get_pixel(y, x), bitmask)
+            
+
+
```

Comment:
Can you run black formatting on this change (`black .`). This will pick up some style guides which will be checked for in the presubmit tests.

Reply:
Ran formatting. Is there any reason we don't configure [husky](https://typicode.github.io/husky/) to run formatting before commits?